<?xml version="1.0"?>
<launch>
    <rosparam file="$(find mavros_test)/config/uav.yaml" />
    <param name="pkg_path" value="$(find mavros_test)/" />

    <!-- Control -->
    <node name="main" pkg="mavros_test" type="main.py" output="screen" />
    <node name="object_detector" pkg="mavros_test" type="object_detector.py" output="screen" />
    <node name="landing_pose_transformer" pkg="mavros_test" type="landing_pose_transformer.py" output="screen" />
</launch>